app/jobs/people/cleanup_job.rb

```diff
@@ -9,12 +9,10 @@ class People::CleanupJob < RecurringJob
   run_every 1.day
 
   def perform_internal
-    people = People::CleanupFinder.new.run
+    return unless FeatureGate.enabled?('people.cleanup_job')
```

Comment:
Das Feature ist by default bereits durch extrem lange Cutoff-Zeiten effektiv deaktiviert. Bitte kontrolliere noch, in welchen Wagons kürzere Zeiten eingestellt sind und aktiviere den neuen Feature Toggle dort noch.

Reply:
Die "extrem lange Cutoff-Zeit" hatte einen Fehler. Noch ist es nirgends eingestellt, das Feature sollte explizit noch nicht aktiviert werden. 

Im Insieme-Wagon habe ich es ebenfalls explizit ausgeschaltet. Wenn wir das Feature allgemein einschalten sollten, ist dort noch der jetzige Kundenwunsch eingestellt.